### PR TITLE
Common Refactor

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,11 @@
     "e2e:publish": "./scripts/e2e-publish.sh",
     "e2e:resolutions": "node ./scripts/e2e-resolutions.js",
     "examples": "npm run examples --workspaces --if-present",
+    "lint": "npm run lint --workspaces --if-present",
+    "lint:fix": "npm run lint:fix --workspaces --if-present",
+    "test": "npm run test --workspaces --if-present",
+    "test:node": "npm run test:node --workspaces --if-present",
```
Comment (acolytec3) [inline package.json:18]: I'm fully in support of these but should they be part of this PR?  If so, can we also update devp2p so it has a `test:node` script as an alias of `test`?

**Review by acolytec3 — COMMENTED** [review]
Looks great so far!  Left a couple of questions for us to confirm before we merge.

File: package.json (hunk 1 above)
Comment (holgerd77) [reply package.json:18]: I would pledge yes, this is such a tiny side change not affecting anything apart of having a bit more comfort. I will add the alias.

File: packages/common/src/common.ts [hunk 2]
```diff
@@ -410,95 +386,88 @@ export class Common {
...
-        ) {
-          value = (hfChanges[1] as any)[topic][name]
+        const hfValue = hfChanges[1].params?.[name]
+        if (hfValue !== undefined) {
+          value = hfValue
         }
       }
       if (hfChanges[0] === hardfork) break
     }
+    if (value === undefined) {
```
[the remark this answers is not in the document]
Comment (holgerd77) [reply packages/common/src/common.ts:430]: Stumbled upon this too, so `value == null` is officially part of the mechanism/logic, meaning that a value from a former HF/EIP is erased again, this is being used e.g. in `netSstoreNoopGas: null, // Removed along EIP-1283`.

Then this goes through here and along `return BigInt(value ?? 0)` below the `null` value gets transitioned into `0`.

**Review by acolytec3 — APPROVED** [review]

**Comment by jochem-brouwer** [comment]
Could the PR description be updated? This is handy for when we write the breaking release notes later :smile: :+1:

**Comment by holgerd77** [comment]
@jochem-brouwer you're right, have done!

**Comment by jochem-brouwer** [comment]
The description is still empty for me though?

**Pull request merged**